deploy.yaml

```diff
@@ -8,23 +8,25 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      name: enrober
+      component: enrober
   template:
     metadata:
       labels:
-        name: enrober
+        component: enrober
         routable: "true"
       annotations:
+        projectcalico.org/policy: "allow tcp from cidr 192.168.0.0/16; allow tcp from cidr 10.129.0.0/16"
         publicHosts: "test.k8s.local"
         publicPaths: "9000:/beeswax/deploy"
```

Comment:
I have a few examples in the `k8s-router` repo that serve the purpose of showing different modes in which you could deploy things.  If you don't need that and this was really just a way for us to deploy `enrober` for our needs, I agree that this could likely go away...unless it's still useful for local environments?